less/about/preferences.less

```diff
@@ -1134,17 +1144,6 @@ table.sortableTable {
       }
     }
 
-    span {
```

Comment:
Note: this is outside of .walletBar. The commit message will be fixed when squashing.